From 387e7d3fe1a86c7544dedac381f0e9787cfb9750 Mon Sep 17 00:00:00 2001
From: Andrew Davis <afd@ti.com>
Date: Fri, 7 Feb 2025 11:08:58 +0530
Subject: [PATCH 12/68] FROMLIST: dt-bindings: soc: ti:
 ti,j721e-system-controller: Add PCIe ctrl property

Add a pattern property for pcie-ctrl which can be part of this controller.

Link: https://lore.kernel.org/all/20241016233044.240699-2-afd@ti.com/
Signed-off-by: Andrew Davis <afd@ti.com>
Reviewed-by: Siddharth Vadapalli <s-vadapalli@ti.com>
---
 .../bindings/soc/ti/ti,j721e-system-controller.yaml          | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Documentation/devicetree/bindings/soc/ti/ti,j721e-system-controller.yaml b/Documentation/devicetree/bindings/soc/ti/ti,j721e-system-controller.yaml
index 378e9cc5fac2..2a64fc61d126 100644
--- a/Documentation/devicetree/bindings/soc/ti/ti,j721e-system-controller.yaml
+++ b/Documentation/devicetree/bindings/soc/ti/ti,j721e-system-controller.yaml
@@ -68,6 +68,11 @@ patternProperties:
     description:
       The node corresponding to SoC chip identification.
 
+  "^pcie-ctrl@[0-9a-f]+$":
+    type: object
+    description:
+      This is the PCIe control region.
+
 required:
   - compatible
   - reg
-- 
2.47.2

